components/molecule/tabs/src/index.scss

```diff
@@ -132,7 +142,7 @@ $c-sui-molecule-tabs: color-variation($c-gray, 1) !default;
       }
 
       &:hover:not(.is-disabled) {
-        color: $c-primary;
+        color: $c-sui-molecule-tabs-disabled;
```

Comment:
Right now, by default this value is changing to `$c-gray-lightest`, is it accepted?

Reply:
Oops, my mistake! 😞

Reply:
Fixed!